v3/src/components/case-table/collection-table-spacer.tsx

```diff
@@ -116,7 +116,7 @@ export const CollectionTableSpacer = observer(function CollectionTableSpacer(pro
   const topButtonTooltip = t(topTooltipKey)
 
   return (
-    <>
+    <div className="collection-table-spacer-wrapper">
```

Comment:
Interesting that this div needs to exist but has no CSS associated with it. 🤔 I would use the simpler class name here that corresponds to the component name:
```suggestion
    <div className="collection-table-spacer">
```